src/main/java/com/powsybl/openloadflow/ac/outerloop/AcLoadFlowParameters.java

```diff
@@ -145,4 +150,8 @@ public boolean isComputeMainConnectedComponentOnly() {
         return computeMainConnectedComponentOnly;
     }
 
+    public Set<Country> getCountriesToBalance() {
+        return countriesToBalance;
```

Comment:
Maybe add `Collections.unmodifiableSet()` to avoid unwanted changes to parameters.